docs/source/markdown/options/time.md

```diff
@@ -5,3 +5,4 @@
 #### **--time**, **-t**=*seconds*
 
 Seconds to wait before forcibly stopping <<the container|running containers within the pod>>.
+Use -1 for infinite wait.
```

Comment:
This options is used by podman pod rm, pod stop, restart, rm, stop based on the auto generated comment.

But you only changed rm and stop, thus the doc is no longer correct.

Reply:
I think this comment still applies.

Reply:
no, looks like all the files in cmd/podman were changed to accept int